decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb

```diff
@@ -80,6 +81,14 @@ def attributes
             announcement: form.announcement
           }
         end
+
+        def related_processes
+          @related_processes ||= Decidim::ParticipatoryProcess.where(id: form.related_process_ids)
+        end
+
+        def link_related_processes
+          @participatory_process.link_participatory_spaces_resources(related_processes, "related_processes")
```

Comment:
the same as in the `create_participatory_process.rb`, should the method name be in singular? `process.link_participatory_space_resources`

Reply:
Good catch, thanks!